examples/twittermap/web/public/javascripts/lib/L.TileLayer.MaskCanvas.js

```diff
@@ -0,0 +1,189 @@
+L.TileLayer.MaskCanvas = L.TileLayer.Canvas.extend({
```

Comment:
we should include the license if this is a third-party lib.

Reply:
I think Jianfeng's comment applies to all external library files that you are adding.